config/jobs/kubernetes-sigs/cluster-api-operator/cluster-api-operator-periodics-main.yaml

```diff
@@ -12,7 +12,7 @@ periodics:
     path_alias: sigs.k8s.io/cluster-api-operator
   spec:
     containers:
-    - image: gcr.io/k8s-staging-test-infra/kubekins-e2e:v20230703-e6ae5b372a-1.25
+    - image: gcr.io/k8s-staging-test-infra/kubekins-e2e:v20230703-e6ae5b372a-1.27
```

Comment:
I was looking into CI today and seems the latest image hash used by k8s infra bot was https://github.com/kubernetes/test-infra/commit/4478ba6eba5fdd7c55b40461e0cc551fe380b223#diff-3718f9c29519d09091e8ebebfca99ec931369c6a540d2cbca4e53cd61f7bb500R19 4 weeks ago

Reply:
you are right... I updated the hash accordingly.